R/ggplot.R

```diff
@@ -1247,7 +1247,7 @@ autoplot.mts <- function(object, colour=TRUE, facets=FALSE, ...){
     
     #Initialise ggplot object
     mapping <- ggplot2::aes_(y=~y, x=~x, group=~series)
-    if(colour){
+    if (colour & (!facets | !missing(colour))){
```

Comment:
That's the logic behind if they've specified the colour argument or not.

Expanding the logic gives: 
`colour & !missing(colour)` (They've specified colour=TRUE as an argument)
`colour & !facets` (They have specified facets=FALSE (or left it as default))

As a result, if facets is TRUE, and if colour is unspecified, then the colour isn't added.